src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,90 @@
+package duke.task;
+
+/**
+ * Represents a task.
+ *
+ * @author Pearlynn
+ */
+
+public abstract class Task {
+
+    /**
+     * The description of the task.
+     */
```

Comment:
I noticed the same issue in several other places too regarding the coding standard for variable commenting